Lib/fortran/fortran.swg

```diff
@@ -0,0 +1,116 @@
+/* -------------------------------------------------------------------------
+ * fortran.swg
+ * ------------------------------------------------------------------------- */
+
+/* -------------------------------------------------------------------------
+   %fortranbegin : Insertion directive to add wrapper code at the very top of
+   the Fortran proxy file.
+
+     Example:
+         %fortranbegin %{
+         ! Auto-generated by SWIG
+         ! Requires foo and bar
+         %}
+
+   %proxycode : Insert Fortran code directly into the `contains` portion of a
+     module. Note that to make the code accessible you will have to add it to the
+     type declartion if appropriate (using `%insert("ftypes")`) or declare it
+     as a public module subroutine (using `%insert("fpublic")`).
+
+   %fortranprepend : Add code at the beginning of a proxy function
+   %fortranappend  : Add code at the end of a proxy function
+
+   %parameter : Force a global constant to be treated as a compile-time value
+           (value MUST be interpretable by Fortran)
+   %enumerator : Force an enum to be treated as a compile-time Fortran enum
+           (enum values MUST be interpretable by Fortran)
+   %bindc : Generate *interfaces only*, no wrapper code, for a function
+ */
+#define %fortranbegin         %insert("fbegin")
+#define %proxycode            %insert("fwrapper")
+
+#define %fortranprepend       %feature("fortran:prepend")
+#define %clearfortranprepend  %feature("fortran:prepend","")
+
+#define %fortranappend        %feature("fortran:append")
+#define %clearfortranappend   %feature("fortran:append","")
+
+
+#define %parameter      %feature("fortran:parameter")
+#define %noparameter    %feature("fortran:parameter","0")
+#define %clearparameter %feature("fortran:parameter","")
+
+#define %enumerator      %feature("fortran:enumerator")
+#define %noenumerator    %feature("fortran:enumerator","0")
+#define %clearenumerator %feature("fortran:enumerator","")
+
+#define %bindc      %feature("fortran:bindc")
+#define %nobindc    %feature("fortran:bindc", "0")
+#define %clearbindc %feature("fortran:bindc", "")
+
```

Comment:
The above should be %fortranbindc, %nofortranbindc, %clearfortranbindc. Similarly for %parameter and %enumerator etc, but I believe these are now going to be %fortranconst instead.

Reply:
Done.